math/binary/logarithm.go

```diff
@@ -0,0 +1,18 @@
+// author(s) [red_byte](https://github.com/i-redbyte)
+// see logarithm_test.go
+
+package binary
+
+// Log_2 Finding the exponent of n = 2**x using bitwise operations (logarithm in base 2 of n) [See more](https://en.wikipedia.org/wiki/Logarithm)
+func Log_2(n uint32) uint32 {
```

Comment:
Don't use underscores in function names. Rename function to `Log2` or `LogBase2`.

Reply:
Done